modules/sortableBidAdapter.js

```diff
@@ -0,0 +1,143 @@
+import * as utils from 'src/utils';
+import { registerBidder } from 'src/adapters/bidderFactory';
+import { config } from 'src/config';
+import { BANNER } from 'src/mediaTypes';
+import { REPO_AND_VERSION } from 'src/constants';
+
+const BIDDER_CODE = 'sortable';
+const SERVER_URL = 'c.deployads.com';
+const SORTABLE_ID = config.getConfig('sortableId');
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [BANNER],
+
+  isBidRequestValid: function(bid) {
+    const haveSiteId = !!config.getConfig('sortableId');
```

Comment:
In lieu of the feedback provided on the docs PR (https://github.com/prebid/prebid.github.io/pull/872); this may need to be updated.